docs/book/orders/orders.rst

```diff
@@ -85,6 +85,15 @@ In order to change the amount of items use the **OrderItemQuantityModifier**.
 
     $this->container->get('sylius.order_item_quantity_modifier')->modify($orderItem, 3);
 
+You can also change **maximum order item quantity** parameter in ``config/services.xml``.
+
+.. code-block:: xml
+
+    # config/services.xml
+    <parameters>
+        <parameter key="sylius.order_item_quantity_modifier.limit">int</parameter> # by default it is 9999
```

Comment:
I think it would be better to give a particular example here:
```suggestion
        <parameter key="sylius.order_item_quantity_modifier.limit">22222</parameter> # by default it is 9999
```